index.js

```diff
@@ -28,10 +33,17 @@ exports.register = function (server, options, next) {
 
   server.route({
     method: 'GET',
-    path: '/dojos/{country}/{path*}',
-    handler: {
-      file: {
-        path: __dirname + '/dist/index.html'
+    path: '/dojos/{id}/{alpha2*}',
```

Comment:
The only thing that's looking at the names of these params is the dojo preloader, so I just changed it to match what it expects.